n2kMappings.js

```diff
@@ -148,7 +184,196 @@ exports.mappings = {
     filter: function(n2k) {
       return n2k.fields['Engine Instance'] === 'Dual Engine Starboard';
     }
-  }],
+  }, {
+    source: 'Oil pressure',
+    node: 'propulsion.port.oilPressure',
+    filter: function(n2k) {
+      return n2k.fields['Engine Instance'] === 'Single Engine or Dual Engine Port';
+    }
+  }, {
+    source: 'Total Engine hours',
+    node: 'propulsion.port.runTime',
+    filter: function(n2k) {
+      return n2k.fields['Engine Instance'] === 'Single Engine or Dual Engine Port';
+    }
+  }, 
+   {
+    node: 'notifications.checkEngine',
```

Comment:
Yeah, there's got to be a better way to handle this. My code that does this for CANboat is three lines long with an array of alarm names. Loop though the names and check the bit index. IF i was better with js, I could probably fix it...